src/content.js

```diff
@@ -126,6 +126,33 @@ function content(target, callback) {
   }
 }
 
+function globals(presentedUrl, contentDoc, callback) {
+    // Fetch globals. Right now the only global is a per-prefix ToC, so this
+    // function is super-simple.
+    var tocUrl = presentedUrl.substring(
+        0,
+        presentedUrl.indexOf(
+            contentDoc.prefix,
+            presentedUrl.indexOf('://') + 3
+        ) + contentDoc.prefix.length
```

Comment:
The Travis build is failing on this line :point_up: I'm guessing that the `contentDoc` I'm setting in nock is missing the `prefix` attribute.